libs/ngworker/lumberjack/ng-package.json

```diff
@@ -1,6 +1,6 @@
 {
   "$schema": "../../../node_modules/ng-packagr/ng-package.schema.json",
-  "dest": "../../../dist/libs/ngworker/lumberjack",
+  "dest": "../../../apps/lumberjack-app/node_modules/@ngworker/lumberjack",
```

Comment:
My only problem with this is that it is non-standard.

If we want to use something like https://github.com/jscutlery/semver and https://www.npmjs.com/package/ngx-deploy-npm this could be a problem. I need to check how they solve the output for creating the release.

Reply:
Jest fails to run if we add the `dist` paths to `tsconfig.base.json` and build the library. This is a workaround for that. Let me know if you find a better alternative that isn't complicated.

Reply:
Let's cross this bridge when we get there.

Reply:
The current solution in this PR breaks the dependency graph between lumberjack-app and the libraries in this workspace so I'm exploring alternatives again.